response/response.go

```diff
@@ -116,14 +116,19 @@ func (res *Response) UnmarshalCBOR(data []byte) error {
 		res.Attestation = possiblyMap.Attestation
 		res.GetRandom = possiblyMap.GetRandom
 		res.Error = possiblyMap.Error
+
+		return nil
```

Comment:
Hey @NullHypothesis, thought you wanted to know... this was the issue with the decoding. It appears I've fixed it on my machine but never really pushed the change. It should be OK now!